eox_nelp/openedx_filters/xapi/filters.py

```diff
@@ -51,3 +54,47 @@ def run_filter(self, result):  # pylint: disable=arguments-differ
         return {
             "result": result
         }
+
+
+class XApiBaseEnrollmentFilter(PipelineStep):
+    """This filter is designed to modify object attributes of an enrollment event, this will add
+    the description field and will change the name based on the course language.
+
+    How to set:
+        OPEN_EDX_FILTERS_CONFIG = {
+            "event_routing_backends.processors.xapi.enrollment_events.base_enrollment.get_object": {
+                "pipeline": ["eox_nelp.openedx_filters.xapi.filters.XApiBaseEnrollmentFilter"],
+                "fail_silently": False,
+            },
+        }
+    """
+
+    def run_filter(self, result):  # pylint: disable=arguments-differ
+        """Modifies name and description attributes of the activity's definition.
+
+        Arguments:
+            result <Activity>: Object activity for events related to enrollments.
+
+        Returns:
+            Activity: Modified activity.
+        """
+        course_id = extract_course_id_from_string(result.id)
+
+        if course_id:
+            # Get course course data
+            course = get_course_from_id(course_id)
+            display_name = course['display_name']
+            description = course['short_description']
+            course_language = course['language']
+
+            # Create new attributes based on the course properties
+            definition_name = LanguageMap(**({course_language: display_name} if display_name is not None else {}))
```

Comment:
Here you are validating `display_name` is not None, but also is possible that `course_language` is empty `None`
![Peek 2023-11-20 13-18](https://github.com/eduNEXT/eox-nelp/assets/51926076/bb6fa943-4d3b-4ee3-9189-e2bbd4522563)

Reply:
@johanv26 How did you set the course language to None since that option is not available on studio ?

Reply:
Actually is a very good question. 
I tried to defined here, but is not working
![image](https://github.com/eduNEXT/eox-nelp/assets/51926076/950f61f6-7688-41cb-a9f6-49aa0ba1f4de)

But on the other hand my course_overview record of that object is empty.
From admin is difficult to change.
![Peek 2023-11-20 16-41](https://github.com/eduNEXT/eox-nelp/assets/51926076/f603d298-b682-4aa6-9d35-60df89b032d5)

Myabe there is a mismatch of mongo and mysql course_overview...